packages/fxa-content-server/app/scripts/views/sign_in_token_code.js

```diff
@@ -6,9 +6,11 @@
  * Allow the user to unblock their signin by entering
  * in a verification code that is sent in an email.
  */
```

Comment:
from meeting: In the cleanup phase, lets look at renaming "token code" to something more meaningful.